frontend/pages/hosts/HostDetailsPage/HostDetailsPage.jsx

```diff
@@ -381,6 +436,7 @@ export class HostDetailsPage extends Component {
         </div>
         {renderLabels()}
         {renderPacks()}
+        {renderSoftware()}
```

Comment:
```suggestion
        {host.software && renderSoftware()}
```
Initially, the Software inventory feature will be behind a feature flag. Since some users may never turn this feature flag on, it makes sense to not render the "Software" table at all for these users. This is a whoops from me as this detail wasn't outlined in the Figma mockups.